code/drasil-lang/lib/Language/Drasil/Space.hs

```diff
@@ -80,9 +86,17 @@ data RealInterval a b where
 -- | Gets the name of an 'Actor'.
 getActorName :: Space -> String
 getActorName (Actor n) = n
-getActorName _ = error "getActorName called on non-actor space"
+getActorName _         = error "getActorName called on non-actor space"
 
 -- | Gets the inner 'Space' of a vector.
 getInnerSpace :: Space -> Space
 getInnerSpace (Vect s) = s
-getInnerSpace _ = error "getInnerSpace called on non-vector space"
+getInnerSpace _        = error "getInnerSpace called on non-vector space"
+
+isNumericSpace :: Space -> Bool
+isNumericSpace Integer  = True
+isNumericSpace Rational = True
+isNumericSpace Real     = True
+isNumericSpace Natural  = True
+isNumericSpace Radians  = True
+isNumericSpace _        = False
```

Comment:
In general, I tend to avoid catch-all for functions like these. This forces people to think things through. For example, maybe `getInnerSpace` above should also work for `Matrix` ?

Reply:
That makes sense. Thank you! Regarding the Matrix and vectors, this function is more of a "isPrimitiveSpaceAndNumeric" (i.e., only basic flat numeric types).

Reply:
Fixed.